bios/lindsay.md

```diff
@@ -1,13 +1,13 @@
-# Jen Reiher
+# Lindsay Macfarlane
 ![Picture of Jen](https://www.canadalearningcode.ca/wp-content/uploads/2017/07/jen.jpg)
```

Comment:
this will still show a picture of me! do you have a picture of you online somewhere? if not delete this line.